libcaf_net/caf/detail/rfc6455.hpp

```diff
@@ -59,7 +59,8 @@ struct CAF_NET_EXPORT rfc6455 {
   static ptrdiff_t decode_header(const_byte_span data, header& hdr);
 
   static constexpr bool is_control_frame(uint8_t opcode) noexcept {
-    return opcode != text_frame && opcode != binary_frame;
+    return opcode != text_frame && opcode != binary_frame
+           && opcode != continuation_frame;
```

Comment:
At this point, wouldn't it make more sense to implement it this way?

```suggestion
    return opcode > binary_frame;
```